app/helpers/search_helper.rb

```diff
@@ -9,7 +9,7 @@ def format(timestamp)
   end
 
   def datafile_formats_for_select
-    Dataset.datafile_formats.unshift('').uniq
+    Dataset.datafile_formats.map(&:downcase).uniq.sort.unshift('')
```

Comment:
ElasticSearch natively sorts that by number of results, but ok why not :)

Reply:
Need to update the description of the spec (`it 'returns a list of unique datafile formats ordered by datafile count (elasticsearch default)`)

Reply:
- It wasn't sorting anything before I added the sort.
- will do, thanks.